net-mail/automx2/automx2-2022.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9,10} )
```

Comment:
Can we clean the unsupported implementations out?

Reply:
Why not. The above list is just what I use for [CI testing](https://github.com/rseichter/automx2/actions/runs/2070824930).